base/namedtuple.jl

```diff
@@ -241,6 +241,18 @@ merge(a::NamedTuple{()}, b::NamedTuple) = b
 
 merge(a::NamedTuple, b::Iterators.Pairs{<:Any,<:Any,<:Any,<:NamedTuple}) = merge(a, b.data)
 
+"""
+    merge(a::NamedTuple, b::NamedTuple, cs::NamedTuple...)
+
+Perform a left-associative merge of three or more named tuples.
```

Comment:
Why not two?

Reply:
Because regular `merge(a::NamedTuple, b::NamedTuple)` already handles the two-argument case? I wrote the docstring for just this single method, which accepts only three or mpre arguments. Should I have done it differently?

Reply:
Maybe we can combine the two docstrings then?

Reply:
Bump.

Reply:
What's the best way to merge the docstrings, just copy one of them into the other after 2 newlines and leave this new `merge` definition without its own docstring? If so, is it ok if I replace `merge(a::NamedTuple, b::NamedTuple)` definition in the docstring with `merge(a::NamedTuple, bs::NamedTuple...)`, even though none of the individual methods have that exact signature?